Formula/t/tcl-tk.rb

```diff
@@ -61,6 +61,13 @@ class TclTk < Formula
       url "https://raw.githubusercontent.com/macports/macports-ports/db4f8f774193/x11/tk/files/fix-themechanged-error.patch"
       sha256 "2a75496dc597dec9d25401ab002f290be74d4acd5566793c5114e75a154c280a"
     end
+
+    # Bugfix for KVO crash; will be in Tk 8.6.14
+    # See https://core.tcl-lang.org/tk/info/310c74ecf4
+    patch :p0 do
+      url "https://raw.githubusercontent.com/macports/macports-ports/6a93695d61d3/x11/tk/files/fix-kvo-crash.diff"
+      sha256 "ec9a9234b4a326e5621fe78e078c29aa4784b6dc88c59a43d828639ebae0af41"
+    end
   end
 
   resource "itk4" do
```

Comment:
Not too sure about this one, but let's see if a mirror helps, like we do for the `tk` resource:

```
mirror "https://src.fedoraproject.org/lookaside/extras/itk/itk4.1.0.tar.gz/sha512/1deed09daf66ae1d0cc88550be13814edff650f3ef2ecb5ae8d28daf92e37550b0e46921eb161da8ccc3886aaf62a4a3087df0f13610839b7c2d6f4b39c9f07e/itk4.1.0.tar.gz"
```

Reply:
It seems the error encountered for the main URL is fatal and does not lead to retrying with the mirror.

Reply:
Yea. But the bottles were built, so it's fine to merge this.

Still, would be nice to figure that out on `brew`'s side, though.